certification/README.md

```diff
@@ -0,0 +1,2 @@
+# certification
+Software Carpentry certifications
```

Comment:
If this comes from our other repository, could you add a link? We may also need to think about updating this dependency (git submodules anyone? @gvwilson thoughts?)